src/Microsoft.Framework.PackageManager/Restore/NuGet/NuGetv2Feed.cs

```diff
@@ -170,6 +175,16 @@ public PackageInfo BuildModel(string id, XElement element)
             var idElement = properties.Element(_xnameId);
             var titleElement = element.Element(_xnameTitle);
 
+            var publishElement = properties.Element(_xnamePublish);
+            if (publishElement != null)
+            {
+                DateTime publishDate; 
+                if (DateTime.TryParse(publishElement.Value, out publishDate) && (publishDate == _unlistedPublishedTime))
```

Comment:
Can you make `IsPublished` a property of the `PackageInfo` class? It might be useful for other purposes, not just for this check